src/main/scala/io/iohk/ethereum/db/dataSource/DataSource.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.db.dataSource
+
+trait DataSource {
+
+  type Key = Array[Byte]
+  type Value = Array[Byte]
+
+  /**
+    * This function obtains the associated value to a key, if there exists one.
+    *
+    * @param namespace which will be searched for the key.
```

Comment:
The main reason is that `IODB` requires all the keys to be of the same size, so as a single `Byte` is enough (at least for now) I decided `namespace: Byte`. If it is preferable I can change it or at least add a few words on this explanation.